package-lock.json

```diff
@@ -1,21 +1,9 @@
 {
   "name": "@okta/okta-signin-widget",
-  "version": "2.2.0",
+  "version": "2.4.0",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
-    "@okta/okta-auth-js": {
```

Comment:
Why is authJs being removed from the dependencies?

Reply:
weird, dunno. I'll rebuild the package-lock.json